lib/src/delegates/asset_picker_text_delegate.dart

```diff
@@ -41,7 +42,8 @@ AssetPickerTextDelegate assetPickerTextDelegateFromLocale(Locale? locale) {
   }
 
   final matchedByScript = scriptCode != null
-      ? matchedByLanguage.where((e) => e.scriptCode == scriptCode)
+      ? matchedByLanguage
+          .where((e) => e.scriptCode?.toLowerCase() == scriptCode)
```

Comment:
Thanks for the catch! Maybe using the lower-cased comparison is a mistake from the start. Could you remove all `toLowerCase` in the code?